scripts/webpack/webpack.hot.js

```diff
@@ -47,25 +47,31 @@ module.exports = merge(common, {
   module: {
     rules: [
       {
-        test: /\.tsx?$/,
+        test: /\.(j|t)sx?$/,
```

Comment:
@davkal what's the reasoning behind this change?

Reply:
I wanted the config to be the same as in react-hot-reloader README in case I overlooked something. `jsx?` won't be included in this PR. Thanks for highlighting it out though.